index.html

```diff
@@ -350,10 +353,10 @@ <h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
                 <input type="submit" value="Get It Touch" class="btn fw-500 fz-17 lh-24" />
               </li>
             </ul>
-            
+
+  
```

Comment:
- [ ] Please not that on submitting the form a method unsupported error is being rendered 😔, kindly make sure your method="POST" instead of post.👍🏾

Reply:
really dont understand what u mwan, can you be more specific